doc/networks.md

```diff
@@ -355,6 +355,17 @@ lxc ls
 +------+---------+---------------------+----------------------------------------------+-----------+-----------+
 ```
 
+Network forwards:
+
+OVN networks support [network forwards](network-forwards.md).
+
+The allowed listen addresses are those that are defined in the uplink network's `ipv{n}.routes` settings, and the
+project's `restricted.networks.subnets` setting (if set).
+
+The listen address used cannot overlap with a subnet that is in use with another network.
+
```

Comment:
Feels a bit weird to have this be in `networks.md`. Given that the only two types that can actually handle forwards are bridged and ovn, I think it'd probably be best to cover their specific restrictions directly in the forwards doc.

Reply:
OK will move, but leave the link there to the forwards section.